Bang/PlayingCards/Visitors/ICardTypeVisitor.cs

```diff
@@ -0,0 +1,97 @@
+using Bang.GameEvents.CardEffects;
+using Bang.Weapons;
+
+namespace Bang.PlayingCards.Visitors
```

Comment:
Why all these interfaces are in one file?

Reply:
Fixed